src/globus_sdk/experimental/globus_app/app.py

```diff
@@ -227,16 +227,75 @@ def _initialize_authorizer_factory(self) -> None:
         authorize requests.
         """
 
+    def login(
+        self,
+        *,
+        auth_params: GlobusAuthorizationParameters | None = None,
+        force: bool = False,
+    ) -> None:
+        """
+        Log an auth entity into the app, if needed, storing the resulting tokens.
+
+        A login flow will be performed if:
+            * ``auth_params`` are provided,
+            * ``force`` is set to True, or
+            * ``self.login_required()`` evaluates to True.
```

Comment:
In Markdown or RST (I don't remember which, or if both), indented bullet points don't render as expected.

I recommend keeping bullet points at the same indentation level as the surrounding text (but don't know with certainty if that applies to RST here).

Are all of these `or` conditions? I'd prefer to have `or` appear at the end of each line if so.

```suggestion
        A login flow will be performed if:

        * ``auth_params`` are provided, or
        * ``force`` is set to True, or
        * ``self.login_required()`` evaluates to True.
```

Reply:
On point 1, these render appropriately in the generated docs: ([generated docs](https://globus-sdk-python--1041.org.readthedocs.build/en/1041/experimental/globus_app.html#globus_sdk.experimental.globus_app.GlobusApp.login)).

Reply:
For these two unioned conditional lists, I'm just going to rephrase it to not sound like a proper english sentence.


```
A login flow will be performed if any of the following hold true:
  * ``auth_params are provided.
  * ...
```

Reply:
I think that's a good solution 👍